index.html

```diff
@@ -13,6 +13,7 @@
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
+        <i class="fas fa-times"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
```

Comment:
when I click on a mobile menu option, a correct part of the page should be displayed as stated in the requirements, In your case, this is not happening,  Kindly fix this issue